examples/bert/requirements.txt

```diff
@@ -1,2 +1,3 @@
 tensorflow
-tensorboardX>=1.8
\ No newline at end of file
+tensorboardX>=1.8
+hyperopt
```

Comment:
Add comments to each of the dependencies, letting users know they are not all required in order to run this example. 

E.g.,
tensorflow  # used for loading bert official model checkpoint
tensorboardX # used only in bert_classifier_using_executor_main.py
....

Reply:
Added in e175fa6